### ISSUE TITLE
Company logo

### ISSUE THREAD

**Issue by drkarl** [fresh-standard/FRESCA#17]
It would also be good to have an optional field for an image with the logo of the company in the work experience section, also maybe in volunteership and the institution in the education section.

**Comment by hacksalot** [comment]
Agreed. In general, let users associate images with things. Should show up across the board in v0.6.1 or v0.7.0.
